css/style.css

```diff
@@ -6,3 +6,107 @@
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+
+
+ 
+
+ body{
+    background-image: url(../images/celeste.gif);
+    background-repeat: no-repeat;
+    background-size:contain
+}
+
+.main-text{
+    font-size: 130pt;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    font-weight:bolder;
+    color: whitesmoke;
```

Comment:
Only one h1 element is assigned this class. This means that this color assignment will be overridden by your h1:first-child pseudo selector and not applied anywhere on the page.